packages/ng/forms/textarea-input/textarea-input.component.ts

```diff
@@ -0,0 +1,31 @@
+import { ChangeDetectionStrategy, Component, Input, numberAttribute, ViewEncapsulation } from '@angular/core';
+import { InputDirective } from '@lucca-front/ng/form-field';
+import { ReactiveFormsModule } from '@angular/forms';
+import { injectNgControl } from '../inject-ng-control';
+import { NoopValueAccessorDirective } from '../noop-value-accessor.directive';
+
+@Component({
+	selector: 'lu-textarea-input',
+	standalone: true,
+	imports: [InputDirective, ReactiveFormsModule],
+	templateUrl: './textarea-input.component.html',
+	hostDirectives: [NoopValueAccessorDirective],
+	encapsulation: ViewEncapsulation.None,
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TextareaInputComponent {
```

Comment:
should we include auto-height directive by default?
https://material.angular.io/cdk/text-field/overview#automatically-resizing-a-textarea

Reply:
I think it will at least require some discussion.

From a personal point of view, I think it poses more UX problems than it solves. Not to mention the fact that we have something that works well natively.

Reply:
We have around 15-20 usages. We can wait and see if someone really needs it

Reply:
Agree with @GuillaumeNury :)